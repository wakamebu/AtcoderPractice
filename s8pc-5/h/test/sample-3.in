eightsixnineonetwozero
6
869120 100
869120 10
869120 8
869120 6
869120 4
869120 2
