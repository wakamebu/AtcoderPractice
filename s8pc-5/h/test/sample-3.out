eihinieonetwoze
ieonetwoze
onetwoze
etwoze
woze
ze
